actions/registry

```diff
@@ -1,152 +0,0 @@
-    value = action_get(param)
-    if not value and not deletion:
-        key = "registry-create-parameter-{}".format(param)
-        error = "failure, parameter {} is required".format(param)
-        action_set({key: error})
-        param_error = True
-
-    context[param] = value
-
-# Create the dockercfg template variable
-dockercfg = '{"%s": {"auth": "%s", "email": "root@localhost"}}' % \
-            (context['domain'], context['htpasswd-plain'])
-context['dockercfg'] = b64encode(dockercfg.encode()).decode('ASCII')
-
-if param_error:
-    sys.exit(0)
-
-# This one is either true or false, no need to check if it has a "good" value.
-context['ingress'] = action_get('ingress')
-
-# Declare a kubectl template when invoking kubectl
-kubectl = ['kubectl', '--kubeconfig=/root/.kube/config']
-
-# Remove deployment if requested
-if deletion:
-    resources = ['svc/kube-registry', 'rc/kube-registry-v0', 'secrets/registry-tls-data',
-                 'secrets/registry-auth-data', 'secrets/registry-access']
-
-    if action_get('ingress'):
-        resources.append('ing/registry-ing')
-
-    delete_command = kubectl + ['delete', '--ignore-not-found=true'] + resources
-    delete_response = call(delete_command)
-    if delete_response == 0:
-        action_set({'registry-delete': 'success'})
-    else:
-        action_set({'registry-delete': 'failure'})
-
-    sys.exit(0)
-
-kube_control = endpoint_from_flag('kube-control.registry_location.available')
-if kube_control:
-    registry_location = kube_control.get_registry_location()
-    context['registry'] = registry_location
-
-# Creation request
-render('registry.yaml', '/root/cdk/addons/registry.yaml',
-       context)
-
-create_command = kubectl + ['create', '-f',
-                            '/root/cdk/addons/registry.yaml']
-
-create_response = call(create_command)
-
-if create_response == 0:
-    action_set({'registry-create': 'success'})
-
-    # Create a ConfigMap if it doesn't exist yet, else patch it.
-    # A ConfigMap is needed to change the default value for nginx' client_max_body_size.
-    # The default is 1MB, and this is the maximum size of images that can be
-    # pushed on the registry. 1MB images aren't useful, so we bump this value to 1024MB.
-    cm_name = 'nginx-load-balancer-conf'
-    check_cm_command = kubectl + ['get', 'cm', cm_name]
-    check_cm_response = call(check_cm_command)
-
-    if check_cm_response == 0:
-        # There is an existing ConfigMap, patch it
-        patch = '{"data":{"body-size":"1024m"}}'
-        patch_cm_command = kubectl + ['patch', 'cm', cm_name, '-p', patch]
-        patch_cm_response = call(patch_cm_command)
-
-        if patch_cm_response == 0:
-            action_set({'configmap-patch': 'success'})
-        else:
-            action_set({'configmap-patch': 'failure'})
-
-    else:
-        # No existing ConfigMap, create it
-        render('registry-configmap.yaml', '/root/cdk/addons/registry-configmap.yaml',
```

Comment:
I see no other references to `registry-configmap.yaml` so I think it can be removed too. For some reason this one is just in the root folder of the charm, not in templates.